docs/guides/how_to_guides/configuring_data_contexts/how_to_instantiate_a_data_context_on_a_databricks_spark_cluster.rst

```diff
@@ -272,7 +272,7 @@ Additional notes
                     datasource_name="insert_your_datasource_name_here",
                     data_connector_name="insert_your_runtime_data_connector_name_here",
                     batch_data=df,
-                    data_asset_name="IN_MEMORY_DATA_ASSET",
+                    data_asset_name="optionally_insert_your_data_asset_name_here",
```

Comment:
I do not think that `data_asset_name` should be optional.  Otherwise, all connectors would have to have a default value, similar to the `IN_MEMORY_DATA_ASSET` presently attached to the `RuntimeDataConnector`.